advanced_source/usb_semisup_learn.py

```diff
@@ -77,6 +77,8 @@
 # - ``Trainer``: a Trainer class for training and evaluating the
 # algorithm on dataset
 # 
+# Note that a CUDA-enabled backend is required for training with the ``semilearn`` package.
+#
 import semilearn
```

Comment:
Should I add a cell `!pip install semilearn`? Since by default the package is not installed.

Reply:
Do I need to create an RST file for this tutorial? Or could I add those instructions directly in this Python file?

Reply:
you add directly to the python file as a comment like the rest of the text in .rst format.